packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -13,38 +13,86 @@ import 'pointer_router.dart';
 
 /// Signature for listening to [PointerEnterEvent] events.
 ///
-/// Used by [MouseTrackerAnnotation], [Listener] and [RenderPointerListener].
+/// Used by [MouseTrackerAnnotation], [MouseRegion] and [RenderMouseRegion].
 typedef PointerEnterEventListener = void Function(PointerEnterEvent event);
 
 /// Signature for listening to [PointerExitEvent] events.
 ///
-/// Used by [MouseTrackerAnnotation], [Listener] and [RenderPointerListener].
+/// Used by [MouseTrackerAnnotation], [MouseRegion] and [RenderMouseRegion].
 typedef PointerExitEventListener = void Function(PointerExitEvent event);
 
 /// Signature for listening to [PointerHoverEvent] events.
 ///
-/// Used by [MouseTrackerAnnotation], [Listener] and [RenderPointerListener].
+/// Used by [MouseTrackerAnnotation], [MouseRegion] and [RenderMouseRegion].
 typedef PointerHoverEventListener = void Function(PointerHoverEvent event);
 
 /// The annotation object used to annotate layers that are interested in mouse
 /// movements.
 ///
-/// This is added to a layer and managed by the [Listener] widget.
+/// This is added to a layer and managed by the [MouseRegion] widget.
 class MouseTrackerAnnotation {
   /// Creates an annotation that can be used to find layers interested in mouse
   /// movements.
   const MouseTrackerAnnotation({this.onEnter, this.onHover, this.onExit});
 
-  /// Triggered when a pointer has entered the bounding box of the annotated
-  /// layer.
+  /// Triggered when a mouse pointer, with or without buttons pressed, has
+  /// entered the annotated region.
+  ///
+  /// This callback is triggered when the pointer has started to be contained
+  /// by the annotationed region for any reason.
+  ///
+  /// More specifically, the callback is triggered by the following cases:
+  ///
+  ///  * A new annotated region has appeared under a pointer.
```

Comment:
tiny nit: in the rest of the docs, we use a " - " as the beginning of bulleted lists in markdown.  Probably best to be consistent.

Reply:
It's actually the first time that I know we sometimes use " - " as bulleted list. Interestingly we actually use " * " a lot too, for example [InkWell](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/ink_well.dart#L111) or [RangeSlider](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/range_slider.dart#L38), as well as all the "See also" sections.

Reply:
Actually, I've apparently been doing it wrong all my life: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments

Please ignore my suggestion above. :-)